us-medical-insurance-costs-checkpoint.ipynb

```diff
@@ -18,7 +18,7 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #14.        else:
</pre><p>This else statement is no longer needed since it never triggers (only brackets with sufficient data in them are analysed by the edited class method) and can be safely removed.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/laurenashley/US-Medical-Insurance-Costs/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>